packages/salesforcedx-apex-debugger/package.json

```diff
@@ -6,7 +6,7 @@
   "publisher": "salesforce",
   "license": "BSD-3-Clause",
   "engines": {
-    "vscode": "^1.40.0"
+    "vscode": "^1.46.0"
```

Comment:
Most users are on the latest 2-3 versions of VSCode but we do have some still running on versions prior to this. I picked this version since it will be a good way to move users to newer versions of VSCode and it's also not a big change for the project going from 1.40 to 1.51.